Traceback (most recent call last):
  File "d:\programs\anaconda3\envs\jbooks\lib\site-packages\jupyter_cache\executors\utils.py", line 56, in single_nb_execution
    record_timing=False,
  File "d:\programs\anaconda3\envs\jbooks\lib\site-packages\nbclient\client.py", line 1087, in execute
    return NotebookClient(nb=nb, resources=resources, km=km, **kwargs).execute()
  File "d:\programs\anaconda3\envs\jbooks\lib\site-packages\nbclient\util.py", line 74, in wrapped
    return just_run(coro(*args, **kwargs))
  File "d:\programs\anaconda3\envs\jbooks\lib\site-packages\nbclient\util.py", line 53, in just_run
    return loop.run_until_complete(coro)
  File "d:\programs\anaconda3\envs\jbooks\lib\asyncio\base_events.py", line 587, in run_until_complete
    return future.result()
  File "d:\programs\anaconda3\envs\jbooks\lib\site-packages\nbclient\client.py", line 541, in async_execute
    cell, index, execution_count=self.code_cells_executed + 1
  File "d:\programs\anaconda3\envs\jbooks\lib\site-packages\nbclient\client.py", line 832, in async_execute_cell
    self._check_raise_for_error(cell, exec_reply)
  File "d:\programs\anaconda3\envs\jbooks\lib\site-packages\nbclient\client.py", line 740, in _check_raise_for_error
    raise CellExecutionError.from_cell_and_msg(cell, exec_reply['content'])
nbclient.exceptions.CellExecutionError: An error occurred while executing the following cell:
------------------
read_write('/content/rosalind_ini5.txt', '/content/out.txt')
------------------

[1;31m---------------------------------------------------------------------------[0m
[1;31mFileNotFoundError[0m                         Traceback (most recent call last)
[1;32m<ipython-input-11-5f5a5d2e4946>[0m in [0;36m<module>[1;34m[0m
[1;32m----> 1[1;33m [0mread_write[0m[1;33m([0m[1;34m'/content/rosalind_ini5.txt'[0m[1;33m,[0m [1;34m'/content/out.txt'[0m[1;33m)[0m[1;33m[0m[1;33m[0m[0m
[0m
[1;32m<ipython-input-10-c11fcaea4528>[0m in [0;36mread_write[1;34m(in_path, out_path)[0m
[0;32m      1[0m [1;31m# reading and writing on files[0m[1;33m[0m[1;33m[0m[1;33m[0m[0m
[0;32m      2[0m [1;32mdef[0m [0mread_write[0m[1;33m([0m[0min_path[0m[1;33m,[0m [0mout_path[0m[1;33m)[0m[1;33m:[0m[1;33m[0m[1;33m[0m[0m
[1;32m----> 3[1;33m   [1;32mwith[0m [0mopen[0m[1;33m([0m[0min_path[0m[1;33m,[0m[1;34m'r'[0m[1;33m)[0m [1;32mas[0m [0mrf[0m[1;33m:[0m[1;33m[0m[1;33m[0m[0m
[0m[0;32m      4[0m     text = [txt for  num, txt in  enumerate(rf.readlines())
[0;32m      5[0m     if num % 2 != 0]

[1;31mFileNotFoundError[0m: [Errno 2] No such file or directory: '/content/rosalind_ini5.txt'
FileNotFoundError: [Errno 2] No such file or directory: '/content/rosalind_ini5.txt'

